src/effects/native/gatereffect.cpp

```diff
@@ -0,0 +1,236 @@
+#include "effects/native/gatereffect.h"
+
+namespace {
+    // Gain when the gate is open, higher than 1 to still have a decent level
+    constexpr double kMaxGain        = 1.5; 
```

Comment:
I don't understand why this isn't 1. It is unintuitive that this effect would add any gain, at least by default. It makes it fairly likely that a deck that is playing at a good level will clip when this effect is turned on. It also makes it impossible to have a smooth transition between turning the effect on and off because there is a sudden volume jump.